presto-main-base/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -102,12 +102,25 @@ public class TaskManagerConfig
     private Duration highMemoryTaskKillerFrequentFullGCDurationThreshold = new Duration(1, SECONDS);
     private double highMemoryTaskKillerHeapMemoryThreshold = 0.9;
     private boolean enableEventLoop;
-    private Duration slowMethodThresholdOnEventLoop = new Duration(10, SECONDS);
+    private Duration slowMethodThresholdOnEventLoop = new Duration(5, MINUTES);
+    private boolean slowMethodLoggingEnabled;
 
-    @Min(50_000_000L)
+    public boolean getSlowMethodLoggingEnabled()
+    {
+        return slowMethodLoggingEnabled;
+    }
+
+    @Config("task.event-loop-enable-slow-method-logging")
+    public TaskManagerConfig setSlowMethodLoggingEnabled(boolean slowMethodLoggingEnabled)
+    {
+        this.slowMethodLoggingEnabled = slowMethodLoggingEnabled;
+        return this;
+    }
+
+    @Min(10L)
     public long getSlowMethodThresholdOnEventLoop()
     {
-        return slowMethodThresholdOnEventLoop.roundTo(NANOSECONDS);
+        return slowMethodThresholdOnEventLoop.roundTo(SECONDS);
```

Comment:
why not just return nanos here and avoid "SECONDS.toNanos()" call in SafeEventLoop ?